modules/statics/src/coins.ts

```diff
@@ -298,6 +298,7 @@ export const coins = CoinMap.fromCoins([
   erc20('bidl', 'Blockbid Liquidity', 2, '0x5c7ec304a60ed545518085bb4aba156e8a7596f6', UnderlyingAsset.BIDL),
   erc20('bird', 'BirdCoin', 18, '0x026e62dded1a6ad07d93d39f96b9eabd59665e0d', UnderlyingAsset.BIRD),
   erc20('bit', 'BitDAO', 18, '0x1a4b46696b2bb4794eb3d4c26f1c55f9170fa4c5', UnderlyingAsset.BIT),
+  erc20('bnb', 'BNB Tokens', 18, '0xb8c77482e45f1f44de1745f52c74426c631bdd52', UnderlyingAsset.BNB),
```

Comment:
```suggestion
  erc20('bnb', 'BNB Token', 18, '0xb8c77482e45f1f44de1745f52c74426c631bdd52', UnderlyingAsset.BNB),
```

Reply:
@doodles92 Can you confirm this as well ^?

Reply:
Yes this is correct; should be "BNB Token"